src/hevm/hevm-cli/hevm-cli.hs

```diff
@@ -693,7 +695,7 @@ symvmFromCommand cmd = do
   (calldata', cdlen, pathCond) <- case (calldata cmd, sig cmd) of
     -- fully abstract calldata (up to 1024 bytes)
     (Nothing, Nothing) -> do
-      cd <- sbytes256
```

Comment:
was this a correctness issue?

Reply:
Yes, I think so. Calldata would not be initialized more than 256 bytes, instead of 1024 as advertized

Reply:
can we add a test that would cover this?